docs/fastddsgen/pubsub_app/includes/first_app.rst

```diff
@@ -42,13 +32,16 @@ The `-example` option creates an example application, and the CMake files needed
 
             <path-to-Fast-DDS-workspace>/src/fastrtpsgen/scripts/fastrtpsgen.bat -example CMake HelloWorld.idl
 
-        - However, if you have followed the **manual installation**, run:
+        - However, if you have followed the **stand-alone installation**, run:
 
         .. code:: bash
 
             <path-to-Fast-DDS-Gen>/scripts/fastrtpsgen.bat -example CMake HelloWorld.idl
 
-Finally, we compile the generated code.
+Build the Fast DDS application
+********************************
+
+Then, compile the generated code.
```

Comment:
Where do I do this steps? The generated files, do I have to move them somewhere or are they generated in the running dir so I can build directly there? I'd include the creation and use of an example dir in these steps